plasma_framework/test/endToEndTests/FastExit.e2e.test.js

```diff
@@ -250,10 +261,39 @@ contract(
 
                                 expect(actualLCBalanceAfterProcessExit).to.be.bignumber.equal(expectedLCBalance);
                             });
+                            describe('When Alice tries to claim the exit bond back', () => {
```

Comment:
maybe let's add one additional test: alice starts an exit with exit bond of size 1, bond size changes, bob starts exit with new exit bond, etc...

Reply:
sure, noted. Merging this and will add that test in the next one?

Reply:
is rather seems to be a quick change - if you have time, please add it now

Reply:
done `c5d4e11`